docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -63,6 +63,17 @@ You can use these helper functions to extract nodes or relationships by label/re
 | apoc.trigger.propertiesByKey($assignedNodeProperties,'key') | function to filter propertyEntries by property-key, to be used within a trigger statement with $assignedNode/RelationshipProperties and $removedNode/RelationshipProperties. Returns [{old,[new],key,node,relationship}]
 |===
 
+[WARNING]
+====
+Unlike Neo4j up to 4, with Neo4j 5 it's not possible to modify an entity created in phase: "after". 
```

Comment:
The wording of this can be improved. Suggestion:

```
Unlike in Neo4j up to 4, with Neo4j 5 it's not possible to modify an already created entity with via trigger with phase "after". 
```

Reply:
Yes, maybe I should write this clearer. I mean entities "just" created (present in `$createdNode` or `$createdRelationships`)

Reply:
I'm happy to approve provided we amend the wording (can be in another PR if we are in a rush to merge this one)